pkgs/applications/graphics/awesomebump/default.nix

```diff
@@ -1,4 +1,4 @@
-{ lib, stdenv, fetchgit, qtbase, qmake, makeWrapper, qtscript, flex, bison, qtdeclarative }:
+{ mkDerivation, lib, stdenv, fetchgit, qtbase, qmake, makeWrapper, qtscript, flex, bison, qtdeclarative }:
```

Comment:
There's no other uses of `stdenv` in this function, so I guess it could be dropped from the arguments?

Or should it use `stdenv.lib` instead of the `lib` argument?